src/actions/articleAction.js

```diff
@@ -0,0 +1,20 @@
+import { GET_ARTICLES_SUCCESS, GET_ARTICLES_FAILURE } from './actionTypes';
+import http from '../services/http';
+
+export const articleActionResponse = (actionTypes, articles) => ({
+  type: actionTypes,
+  payload: articles
+});
+
+export const getArticles = () => async(dispatch) => {
+  try {
+    const response = await http.get('/articles?size=12');
+    const { data: { data } } = response;
+    if (response.status === 404) {
+      dispatch(articleActionResponse(GET_ARTICLES_FAILURE, 'Article not found'));
```

Comment:
#### Nitpick

- Add Asynchronous caveat ie. `LOADING`

Reply:
Done, thank you